app/views/admin/users/show.html.erb

```diff
@@ -60,6 +60,10 @@
                             <td><strong><%= t('created_at') %></strong></td>
                             <td><%= I18n.l(@user.created_at) %></td>
                         </tr>
+                        <tr>
+                            <td><strong><%= t('users.daily_summary') %></strong></td>
+                            <td><%= @user.daily_summary? ? I18n.t('yes') : I18n.t('no')%></td>
```

Comment:
I would use `t` here, since `I18n.t` is not the same as `t`. 
For example, `config.action_view.raise_on_missing_translations` has no effect when using `I18n.t` (which is bad).